apps/debt/templates/debt/list.html

```
@@ -72,67 +31,75 @@ <h4 class="mb-1">{% trans "Optimised debts" %}</h4>
+            <div class="overflow-x-auto">
+                <table class="align-middle mb-0 w-full">
+                    <thead class="text-text-secondary uppercase text-xs hidden md:table-header-group">
+                        <tr>
+                            <th scope="col">{% trans "Debt" %}</th>
+                            <th scope="col" class="text-right">{% trans "Status" %}</th>
+                        </tr>
+                    </thead>
+                    <tbody>
+                        {% for debt in debts %}
+                            <tr class="{% if forloop.first %}border-t{% endif %} border-b border-gray-200 dark:border-dark-border{% if not debt.settled and debt.debitor.name == request.user.name %} cursor-pointer hover:bg-gray-50 dark:hover:bg-dark-hover{% endif %}"
+                                {% if not debt.settled and debt.debitor.name == request.user.name %} hx-get="{% room_url 'debt:settle' pk=debt.id %}" hx-target="#body" hx-indicator="#body-loading-spinner" hx-swap="morph:innerHTML" hx-push-url="true" {% endif %}>
```

HTMLHint (1.9.2)

[error] Duplicate of attribute name [ {% ] was found.

(attr-no-duplication)